js/filter.js

```diff
@@ -90,24 +90,24 @@ function renderSpeciesPage(speciesData, permalink) {
   var specTax = document.createElement("p");
   var enclosingTaxa = [
     ["Subclass", speciesData.subclass],
-    ["Infraclass", speciesData.infraClass],
+    ["Infraclass", speciesData.infraclass],
```

Comment:
These would be great items to subject to unit tests. Did you ever consider implementing tape unit tests? See test/ folder for example.